### PR TITLE
ODIM-4050 Publish only standard events to NB client

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by bharath-b-hpe — APPROVED** [review]

File: svc-events/events/publishevttosubscriber.go [hunk 1]
```diff
@@ -110,41 +64,50 @@ func PublishEventsToDestination(data interface{}) bool {
...
+			continue
+		}
+
+		if len(inEvent.OriginOfCondition.Oid) < 1 {
+			log.Info("event not forwarded : Originofcondition is empty in incoming event with body: ", requestData)
 			continue
 		}
+		if strings.EqualFold(inEvent.EventType, "ResourceAdded") &&
+			strings.HasPrefix(inEvent.OriginOfCondition.Oid, "/redfish/v1/Fabrics") {
+			addFabricRPCCall(inEvent.OriginOfCondition.Oid, host)
```
Comment (amar-shalgar) [inline svc-events/events/publishevttosubscriber.go:106]: Please cross check with CFM plugin, earlier addFabricRPCCall was not getting called when an fabric added events comes to odimra

**Review by amar-shalgar — COMMENTED** [review]
Please cross check the functionality with CFM plugin

**Review by skhan-anjum — APPROVED** [review]

File: svc-events/events/submittestevent.go [hunk 2]
```diff
@@ -203,7 +203,9 @@ func validAndGenSubTestReq(reqBody []byte) (*ForwardEvent, string, string, []int
 	if val, ok := req["OriginOfCondition"]; ok {
 		switch v := val.(type) {
 		case string:
-			testEvent.OriginOfCondition = v
+			testEvent.OriginOfCondition = &common.Link{
```
Comment (Bharath-KKB) [inline svc-events/events/submittestevent.go:207]: Add comment "As per EventService spec in the SubmitTestEvent subschema OriginOfCondition is a string. However we need to convert this to an object as the event publisher will drop these events. "

File: svc-events/events/publishevttosubscriber.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-events/events/publishevttosubscriber.go:106]: I have tested with event-publisher binary by creating fabric event, `addFabricRPCCall ` method getting called.

File: svc-events/events/submittestevent.go (hunk 2 above)
Comment (irappa-pattar) [reply svc-events/events/submittestevent.go:207]: added comment

**Review by Bharath-KKB — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]
LGTM

**Comment by Bharath-KKB** [comment]
This needs to go to RC0 branch as well.

**Review by jeevan-kamkar — APPROVED** [review]

**Pull request merged**